src/shortcuts.js

```diff
@@ -100,7 +100,7 @@ export const globalShortcuts = perma(() => [
         const itemsRanked = lastItemsFromContextChain(contextChain)
         const children = getChildrenWithRank(itemsRanked)
 
-        if (sigKey(cursor) === '' && children.length === 0) {
+        if ((sigKey(cursor) === '' && children.length === 0) || (sigKey(cursor) === '---' && sigRank(cursor) === 0)) {
```

Comment:
I don't believe we want the `sigRank` check here. We should be able to delete the cursor no matter what rank (order in the list) it has.

Reply:
I added that for the case when a divider is the very first thought. Is there a better place to do that check to delete it?

Reply:
Ah, I see! Is there a reason we need a special case for when the divider is the very first thought? Wouldn't that be handled by the normal case? Sorry if I am misunderstanding..

FWIW, the first thought may not have a rank of 0, as ranks may start at a higher number if the 0 rank thought is deleted, not to mention ranks can be negative.